java/performance/src/org/netbeans/modules/performance/utilities/CommonUtilities.java

```diff
@@ -279,7 +279,7 @@ public static String jEditProjectOpen() {
             ZipEntry entry;
             while ((entry = zis.getNextEntry()) != null) {
                 if (entry.isDirectory()) {
-                    new File(System.getProperty("nbjunit.workdir") + File.separator + ".." + File.separator + "data" + File.separator + entry.getName()).mkdir();
+                    new File(System.getProperty("nbjunit.workdir") + File.separator + ".." + File.separator + "data" + File.separator + entry.getName()).mkdirs();
```

Comment:
This assumes, that the upper levels of the directory where not yet created. The order in the zip is off. If that is the case, it will still blow in line 287, when the content file is to be written to the not existing directory.